src/util/dev/errors.ts

```diff
@@ -0,0 +1,76 @@
+export const httpStatusDescriptionMap = new Map([
+  [400, 'BAD_REQUEST'],
+  [402, 'PAYMENT_REQUIRED'],
+  [403, 'FORBIDDEN'],
+  [404, 'NOT_FOUND'],
+  [405, 'NOT_ALLOWED'],
+  [410, 'GONE'],
+  [413, 'PAYLOAD_TOO_LARGE'],
+  [429, 'RATE_LIMITED'],
+  [500, 'INTERNAL_SERVER_ERROR'],
+  [501, 'NOT_IMPLEMENTED'],
+  [502, 'BAD_GATEWAY'],
+  [503, 'SERVICE_UNAVAILABLE'],
+  [504, 'GATEWAY_TIMEOUT'],
+  [508, 'INFINITE_LOOP']
+]);
```

Comment:
Can these error code be shared with now-proxy or is that currently lua? I wonder if we store as JSON so that lua can parse and now-cli can parse.

Reply:
They were copy/pasted from the Now proxy code, it's in lua.